### ISSUE TITLE
Magic Leap build has deprecation warnings with SDK v0.20.0

### ISSUE THREAD

**Issue by asajeffrey** [servo/servo#23267]
```
$ MLCERT=~/MagicLeap/cert/alanjeffreymlcert.cert MAGICLEAP_SDK=/Users/ajeffrey/MagicLeap/mlsdk/v0.20.0 ./mach package -d --magicleap
[Servo2D] Compiling Servo2D.cpp...
/Users/ajeffrey/github/asajeffrey/servo/support/magicleap/Servo2D/code/src/Servo2D.cpp:279:39: error: 'getServerEventType' is deprecated
      [-Werror,-Wdeprecated-declarations]
  lumin::ServerEventType typ = event->getServerEventType();
                                      ^
/Users/ajeffrey/MagicLeap/mlsdk/v0.20.0/include/runtime/core/lumin/event/ServerEvent.h:78:43: note: 'getServerEventType' has been explicitly marked deprecated here
      INTERGEN_DEPRECATED ServerEventType getServerEventType() const;
                                          ^
1 error generated.
make: *** [/Users/ajeffrey/github/asajeffrey/servo/target/magicleap/aarch64-linux-android/debug/debug_lumin_clang-3.8_aarch64/Servo2D.mk:36: /Users/ajeffrey/github/asajeffrey/servo/target/magicleap/aarch64-linux-android/debug/debug_lumin_clang-3.8_aarch64/obj.Servo2D/code/src/Servo2D.cpp.o] Error 1
Packaging Magic Leap exited with return value 1
```

**Comment by asajeffrey** [comment]
IRC chat with @jdm https://mozilla.logbot.info/servo/20190425#c16245518

TL;DR: suppress the deprecation for now and kick that can down the road.

**Comment by asajeffrey** [comment]
#23268 suppresses the error, but I'm leaving this issue open until we update to v0.20.0 or later of the mlsdk.
